site/public/css/ta-grading.css

```diff
@@ -5,3 +5,4 @@
 .hotkeys-button {
     width: 125px;
 }
+
```

Comment:
This whitespace change isn't related to the rest of the PR and should be reverted.

Reply:
sorry for this mistake, now it's reverted.